terraform/modules/user_groups/main.tf

```diff
@@ -0,0 +1,75 @@
+# user_groups/main.tf
+
+variable "region" {
```

Comment:
good to move this to a variables.tf file, see other modules

Reply:
Okay makes sense, will do